docs/www/nodule-grid.js

```diff
@@ -377,6 +377,368 @@
+                        $(nt).removeClass("update-data");
+                    }
+
+                    // "Radiographic Lesion Margin"
+                    se = "#cect" + noduleId + "se";
+                    sp = "#cect" + noduleId + "sp";
+                    if ($(se).hasClass("update-data")) {
+                        if ($(se).attr("original-value") === "true") {
+                            $(se).prop("checked", true);
+                        } else {
+                            $(se).prop("checked", false);
+                        }
+                        $(se).removeAttr("original-value");
+                        $(se).removeClass("update-data");
+                        // also remove the class "update-data" from the parent's parent which is a div to be used for stlying
+                        //$(se).parent().parent().removeClass("update-data-grandparent");
+                    }
+                    if ($(sp).hasClass("update-data")) {
+                        if ($(sp).attr("original-value") === "true") {
+                            $(sp).prop("checked", true);
+                        } else {
+                            $(sp).prop("checked", false);
+                        }
+                        $(sp).removeAttr("original-value");
+                        $(sp).removeClass("update-data");
+                        // also remove the class "update-data" from the parent's parent which is a div to be used for stlying
+                        //$(sp).parent().parent().removeClass("update-data-grandparent");
+                    }
+
+                    // "Maximum 2D diameter"
+                    sl = "#cect" + noduleId + "sl";
+                    if ($(sl).hasClass("update-data")) {
+                        $(sl).val($(sl).attr("original-value"));
+                        $(sl).removeAttr("original-value");
+                        $(sl).removeClass("update-data");
+                    }
+
+                    // "Maximum perpendicular 2D diameter"
+                    sl = "#cect" + noduleId + "sw";
+                    if ($(sl).hasClass("update-data")) {
+                        $(sl).val($(sl).attr("original-value"));
+                        $(sl).removeAttr("original-value");
+                        $(sl).removeClass("update-data");
+                    }
+
+                    // "Volume"
+                    sv = "#cect" + noduleId + "sv";
+                    if ($(sv).hasClass("update-data")) {
+                        $(sv).val($(sv).attr("original-value"));
+                        $(sv).removeAttr("original-value");
+                        $(sv).removeClass("update-data");
+                        $(sv).parent().removeClass("update-data-parent");
+                    }
+                }
+                $(".revert-field").remove(); // remove all revert-field buttons
+            }
+
+            function _updateData() {
+                // theData is hard coded here for now
+                theData = [
```

Comment:
Move this to a constant for now. Rename to `dicomStructuredReport`. This value should be passed into the function as an argument.